scipy/stats/_boost/meson.build

```diff
@@ -66,6 +66,15 @@ ncx2_ufunc = py3.extension_module('ncx2_ufunc',
   subdir: 'scipy/stats/_boost'
 )
 
+ncf_ufunc = py3.extension_module('nct_ufunc',
```

Comment:
@mckib2 Just found a typo here. `ncf_ufunc` didn't get replaced with `nct_ufunc`.  Are these names important?

Reply:
meson will use that name for the name of the build target, I don't believe it will be used for anything else.  It just needs to be unique, so as long as nothing else in the project is named "nct_ufunc", then we're fine

Reply:
nct_ufunc is a few lines above. Please take a look to see if you would have expected it to cause a problem.

Reply:
@mckib2 I wasn't sure whether this was understood - the question is more about the name of the _variable_ `ncf_ufunc` than the build target name string `'nct_ufunc'`. Here, we accidentally redefined what the variable `ncf_ufunc` refers to - now it refers to the extension for NCT instead of NCF. 

It doesn't look like the variable names are used elsewhere in this module, but perhaps they are imported elsewhere? Does this affect anything? Do we even need the variables to have names at all?

The distributions seem to be working correctly, but I thought I should double-check as we prepare for 1.9.1.